string user

string cpu_model
uint8 cpu_core

string total_mem
string free_mem

uint8 gpu_num
string gpu_model

# inference_speed
float64 preprocess
float64 inference
float64 postprocess

bool is_possible
